src/IBM.Watson.VisualRecognition.v4/VisualRecognitionService.cs

```diff
@@ -348,10 +381,17 @@ public DetailedResponse<Collection> GetCollection(string collectionId)
         /// encoding if it encounters non-ASCII characters.
         /// </summary>
         /// <param name="collectionId">The identifier of the collection.</param>
-        /// <param name="collectionInfo">The updated collection. (optional)</param>
+        /// <param name="name">The name of the collection. The name can contain alphanumeric, underscore, hyphen, and
+        /// dot characters. It cannot begin with the reserved prefix `sys-`. (optional)</param>
+        /// <param name="description">The description of the collection. (optional)</param>
+        /// <param name="trainingStatus">Training status information for the collection. (optional)</param>
```

Comment:
can we remove training status from here

Reply:
fixed in [8b40c5f](https://github.com/watson-developer-cloud/dotnet-standard-sdk/pull/445/commits/8b40c5fad365de2a712a6bbc0b56f8aae59e6626)